src/tr/yaoibar/build.gradle

```diff
@@ -0,0 +1,11 @@
+ext {
+    extName = 'Yaoibar'
+    extClass = '.Yaoibar'
+    themePkg = 'madara'
+    baseUrl = 'https://yaoibar.gay'
+    overrideVersionCode = 0
+    isNsfw = true
+}
+
+apply from: "$rootDir/common.gradle"
+
```

Comment:
Too many trailing newlines.
You can check how many you have with:
```bash
xxd src/tr/yaoibar/build.gradle
```

It should end in one `0a`, not `0a0a` as it is now.

Reply:
IT wasnt showing in the files changed tab on Github so i added another one.